### PR TITLE
cargo-msrv: init at 0.4.0

### PR DESCRIPTION
**Pull request by otavio**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by fufexan — APPROVED** [review]

File: pkgs/development/tools/rust/cargo-msrv/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    updateScript = nix-update-script {
+      attrPath = pname;
+    };
+  };
+
+  doCheck = false;
+
+  buildInputs = [ openssl ];
+
+  nativeBuildInputs = [ makeWrapper pkg-config ];
```
Comment (legendofmiracles) [inline pkgs/development/tools/rust/cargo-msrv/default.nix:33]:
```suggestion
  nativeBuildInputs = [ pkg-config ];
```
I'm not sure if you need `makeWrapper` here. On my system it also works without.

**Review by legendofmiracles — CHANGES_REQUESTED** [review]

File: pkgs/development/tools/rust/cargo-msrv/default.nix (hunk 1 above)
Comment (otavio) [reply pkgs/development/tools/rust/cargo-msrv/default.nix]: Fixed.

**Review by legendofmiracles — APPROVED** [review]

**Comment by legendofmiracles** [comment]
Result of `nixpkgs-review pr 120961` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cargo-msrv</li>
  </ul>
</details>

File: pkgs/development/tools/rust/cargo-msrv/default.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  cargoSha256 = "03rphdps17gzcmf8n5w14x5i5rjnfznsl150s3cz5vzhbmnlpszf";
+
+  passthru = {
+    updateScript = nix-update-script {
+      attrPath = pname;
+    };
+  };
+
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/tools/rust/cargo-msrv/default.nix:28]: Why did you disable tests?

File: pkgs/development/tools/rust/cargo-msrv/default.nix [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  buildInputs = [ openssl ];
+
+  nativeBuildInputs = [ pkg-config ];
+
+  meta = with lib; {
+    description = ''Cargo subcommand "msrv": assists with finding your minimum supported Rust version (MSRV)'';
```
Comment (SuperSandro2000) [inline pkgs/development/tools/rust/cargo-msrv/default.nix:35]:
```suggestion
    description = "Cargo subcommand \"msrv\": assists with finding your minimum supported Rust version (MSRV)";
```

File: pkgs/development/tools/rust/cargo-msrv/default.nix (hunk 2 above)
Comment (otavio) [reply pkgs/development/tools/rust/cargo-msrv/default.nix:29]: Fixed, added a comment.

**Comment by otavio** [comment]
News? @SuperSandro2000

**Review by jonringer — APPROVED** [review]
LGTM
shows usage

```
https://github.com/NixOS/nixpkgs/pull/120961

1 package built:
cargo-msrv
```

**Pull request merged**